### ISSUE TITLE
Ganimation output problem

### ISSUE THREAD

**Issue by SamchooYang** [sh1newu/SepMark#7]
Have you ever encountered this problem when using GANimation's pre-trained model? How can I solve it?
![00044](https://github.com/sh1newu/SepMark/assets/105795555/f5c8a3f4-5d55-4c63-9f12-c3f05b1cd425)
![ZX20uJPsO9oLntCG](https://github.com/sh1newu/SepMark/assets/105795555/ae78ff42-7f66-482e-a6e3-7ef1508a8ebf)

**Comment by sh1newu** [comment]
To our knowledge, the training resolution of GANimation's pre-trained model is 128px (with cropped face images), the official SimSwap's pre-trained model is 224px (by observation it also works on 256px during inference), and StarGAN releases two pre-trained models (128px and 256px). So we conducted experiments at both resolutions of 128px and 256px and did not retrain these pre-trained models. You can try to reproduce and retrain the model at the resolution you prefer, as well as any other generative models, by yourself. Good luck!

**Comment by SamchooYang** [comment]
First of all, thanks for your answer! From the data in Sepmark and the GANimation dataset you provided, it seems that the pretrained model of GANimation supports 256px. Does this means you test 256px images on 128px pre-trained model.

**Comment by sh1newu** [comment]
Yes, you're right. The 128px pre-trained GANimation does support 256px inputs, but the visual results may not be good, as you provided examples. We only take some representative Deepfakes as black box distortions in the noise layer, and the training or retraining of the generative model is somewhat beyond the scope of this work.

**Comment by SamchooYang** [comment]
ok, thanks for your answer! Best wishes!
